skfda/representation/grid.py

```diff
@@ -710,12 +711,11 @@ def __eq__(self, other: object) -> NDArrayBool:  # type: ignore[override]
 
     def _get_op_matrix(
         self,
-        other: Union[T, NDArrayFloat, NDArrayInt, float],
+        other: Union[T, NDArrayFloat, NDArrayInt, float, Callable],
```

Comment:
We shouldn't accept arbitrary callables. Please include the expected input and output types.